
# try: 
#     x=input("enter number")
#     res=float(x)%2

    
# except TypeError: 
#     print("Caught TypeError Exception") 
# else: 
#     print("No exception")

# class age:
#     def (self,x)
#     try:
        
#     raise ValueError("That is not a positive number!") 

#     finally:
#     print('Goodbye, world!')

#         if int(x)%2:
        
# try: 
#     a = int(input("Enter a positive integer: ")) 
#     if a <= 0: 
#         raise ValueError("That is not a positive number!") 

# except ValueError as e: 
#     print(e) 


x=input("enter two number") 


